pxr/usd/usdGeom/schema.usda

```diff
@@ -2292,6 +2292,35 @@ class Camera "Camera" (
     However, it follows that if even one property is authored in the correct
     scene units, then they all must be.
 
+    \\section UsdGeom_CameraExposure Camera Exposure Model
+
+    UsdGeomCamera models exposure by a camera in terms of exposure time, ISO,
+    f-number, and exposure compensation, mirroring the controls on a real camera.
+    These parameters are provided by \\ref UsdGeomCamera::GetExposureTimeAttr(),
+    \\ref UsdGeomCamera::GetExposureIsoAttr(), \\ref UsdGeomCamera::GetExposureFNumberAttr(), 
+    and \\ref UsdGeomCamera::GetExposureAttr(), respectively. 
+    \\ref UsdGeomCamera::GetExposureResponsivityAttr() provides an additional scaling
+    factor to model the overall responsivity of the system, including response of 
+    the sensor and loss by the lens.
+
+    The combination of these parameters gives an exposure scaling factor (also 
+    referred to as the __imaging ratio__) that converts from incident luminance at
```

Comment:
"at sensor" and "output signal" seems to imply that we do not take into affect things that happen before the sensor such as the lens and aperture size. Yet, we include the fStop, so this doesn't seem quite correct.

Reply:
Well the light has to get through the lens to reach the sensor :) I've added wording to clarify